【Celo Gallery】Celo Toolkit - web interface for managing Celo accounts

safepayment opened the pull request:
### Description

Celo Toolkit is the web interface for managing Celo accounts. It will lower the barrier for developers and newbies. Phone number or app downloads are not necessary.

It's [LIVE](http://celo.mugglepay.com/).
[Github](https://github.com/MugglePay/celo-toolkit)

### Tested

No tests. It's a web portal (user interface) to make it easy for new developers and newbies.

### Other changes

No

### Related issues

No.

I've dapp experience on Ethererum, EOS, Tron. When I develop with the current Celo Network SDK, I still found few things frustrated.
1) Celo app cannot be downloaded in my country, and it took me some time to get around this.
2) The process of invitation code and Celo registration is not stable in Asia area.

It took me an hour but it might take more time for developers new to blockchain. So we build the web wallet help the reduce the cost for celo developers or newbies.

### Backwards compatibility

It's a new feature.

Commit:
add web wallet readme

Commit:
add celo dapp gallery

safepayment commented:
@nategraf Would you mind having a look on this?
It's a PR from Celo Camp Community :)

safepayment commented:
Greetings from MugglePay@Celo Camp! We are building a web toolkit with celo sdk. It's open sourced and live. We are contributing to the Celo apps. @prasincs @jeanregisser @mrsmkl @ashishb @nategraf 

Live: http://celo.mugglepay.com/
Github: https://github.com/MugglePay/celo-toolkit

critesjosh commented on packages/docs/README.md:
```diff
@@ -97,6 +97,7 @@ Browse the code, raise an issue, or contribute a PR:
 
 Try Celo out:
 
+- [Using the Web Wallet](getting-started/using-the-web-wallet.md) on the Alfajores Testnet
```
this is not a main net production ready web wallet. so I don't think it should be in the main README page.

critesjosh commented on packages/docs/developer-resources/celo-dapp-gallery.md:
```diff
@@ -2,6 +2,12 @@
 
+## Celo Toolkit
+
+Celo Toolkit is the open source interface for managing Celo accounts on Web. Start with Celo now in 1 minutes. Phone number or app downloads are not necessary.
```
I think it would be helpful to make it more clear on the page that this tool kit is for the TESTNET only. This is not a tool for managing accounts on the mainnet.

critesjosh reviewed (commented):
Can you make it more clear that this is for the Alfajores testnet only? 

Also, I would prefer to limit this PR to just the addition of the app to the Dapp Gallery for now.

safepayment commented:
@critesjosh Thanks for the review.

I've removed the changes in main.md and added only to the Dapp Gallery in the other PR. Close this thread as DUPLICATE.
https://github.com/celo-org/celo-monorepo/pull/3726/files
